Package.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.1
+
+import PackageDescription
+
+let package = Package(
+    name: "Tom",
+    platforms: [
+        .iOS(.v13)
```

Comment:
```suggestion
        .iOS(.v11)
```
Here I added the platform version number incorrectly. Sorry 🙃
cc: @mustafagunes